xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsApplicationsPreferencesMacro.xml

```diff
@@ -1044,7 +1045,12 @@ require(['jquery', 'xwiki-meta', 'ApplicationWidget', 'xwiki-bootstrap-switch',
       #end
       ## Collapse button
       &lt;td class="notificationCollapseButtonCell"&gt;
-        &lt;button type="button" class="btn btn-default btn-sm collapseButton"&gt;$services.icon.renderHTML('caret-down')&lt;/button&gt;
+        &lt;button id="notifications-applications-preferences-macro-$appCount-button" type="button" class="btn btn-default btn-sm collapseButton"&gt;
```

Comment:
Now that there is no more label, you can remove the id again and also the `$appCount` variable.

Reply:
:face_with_spiral_eyes: Trying to follow with fixes too fast, I should take my time and actually do something correctly...

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2217/commits/3f4fdf1ad81aecc0b7af41299cb41b6c3ca60ac6